### ISSUE TITLE
TexText 1.0: "lxml module missing"

### ISSUE THREAD

**Issue by pglpm** [textext/textext#202]
I installed Inkscape 1.0beta and TexText 1.0 as per the [installation page](https://textext.github.io/textext/install/windows-beta.html).

The installation seems to succeed, with a final message

```
[TexText][SUCCESS ]: --> TexText has been SUCCESSFULLY installed on your system <--
```

Now there's a quirk, but it isn't the main problem: I launch Inkscape 1.0beta, and the TexText extension appears *under the `Extensions -> Text` submenu, rather than directly under `Extensions`. But maybe now it's that way by design.

The real problem is that upon launching TexText I get the error message reported below, complaining of a missing `lxml` module. This is strange, because I do see a `C:\ProgramData\inkscapebeta\lib\python3.8\site-packages\lxml\` directory in the Inkscape directory. Maybe this is a problem of Inkscape, not TexText?

Additional info:

* I tried with these three Inkscape distributions (deleting the TexText extension files every time), but the problem persists:
– 1.0beta2 (5ba348d, 2020-03-28)
– 1.0rc1 (09960d6, 2020-04-09)
– 1.1-dev (c990a67, 2020-04-09)

* I have python3.8 and python2.7 distributions on my machine. I tried the installation of Inkscape+TexText including first python3.8 only, then the python2.7 only, then neither, in my PATH environment variable. The problem persists.

* I have also inkscape 0.92.4, as portable.

* Installation of TexText seems to complain about GTK3 not found:

```
[TexText][SUCCESS ]: /-and-+ [Succ] Detect GUI library
[TexText][INFO    ]:       /--or-* [Fail] GTK3 is not found
[TexText][INFO    ]:       |      Please follow installation instructions at
[TexText][INFO    ]:       |             https://textext.github.io/textext/install/windows.html#windows-install-pygtk2
[TexText][SUCCESS ]:       /--or-* [Succ] TkInter is found
```
So I installed `PyGTK-2.24-Inkscape-0.92.4-32bit` on Inkscape 0.94.2 as described in the page mentioned in that output, but the problem persists.

<details>
<summary>dialog output (if available):</summary>

```
[TexText][ERROR   ]: No module named 'lxml'
[TexText][ERROR   ]: Traceback (most recent call last):
  File "__init__.py", line 109, in <module>
    import inkex
  File "C:\ProgramData\inkscapebeta\share\inkscape\extensions\inkex\__init__.py", line 11, in <module>
    from .extensions import *
  File "C:\ProgramData\inkscapebeta\share\inkscape\extensions\inkex\extensions.py", line 33, in <module>
    from .elements import load_svg, BaseElement, ShapeElement, Group, Grid, \
  File "C:\ProgramData\inkscapebeta\share\inkscape\extensions\inkex\elements.py", line 30, in <module>
    from lxml import etree
ModuleNotFoundError: No module named 'lxml'

[TexText][INFO    ]: TexText finished with error, please run extension again
[TexText][INFO    ]: If problem persists, please file a bug https://github.com/textext/textext/issues/new?template=bug_report.md
```
</details>

**Logfile**

Attached
[textext.log](https://github.com/textext/textext/files/4459120/textext.log)

**System information**

- TexText version: TexText 1.0.0-dev.2 (2020-02-10)
- Inkscape version: 1.0beta2 (5ba348d, 2020-03-28) (but see above)
- Operating system: Windows 7 32-bit
- SVG-converter installed (pstoedit+ghostscript or pdf2svg): no idea, sorry!
- GUI framework installed (PyGTK, PyGTK+PyGTK-Sourceview, TkInter): TkInter I think.
- Windows only: LaTeX-distribution (MiKTeX, TeX-Live): TeX-Live
- Windows only: TexText installed via batch script or installer file: Batch

**Comment by pglpm** [comment]
UPDATE: I have installed `lxml` in my python3.8 distribution, and added the latter to my PATH environment variable. Now TexText works. So evidently it isn't using the python distribution of inkscape1.0 – not fully at least.

**Comment by jcwinkler** [comment]
Usually Inkscape is shipped with its own Python environment on Windows which includes lxml. Are you sure that it is included in your installation?

**Comment by pglpm** [comment]
Yes, absolutely sure. I have a subdirectory  
`C:\ProgramData\inkscapebeta\lib\python3.8`  
and I see subdirectories  
`C:\ProgramData\inkscapebeta\lib\python3.8\site-packages\lxml`  
`C:\ProgramData\inkscapebeta\lib\python3.8\site-packages\lxml-4.5.0-py3.8.egg-info`  

Not quite sure why TexText doesn't see them, it seems to see all the rest. I'm trying to see if I have strange entries in my PATH.

Do you mean I should add Inkscape's python distribution to my PATH?

**Comment by sizmailov** [comment]
> Do you mean I should add Inkscape's python distribution to my PATH?

@pglpm No, inkscape should handle it on it's own.

**Comment by pglpm** [comment]
> 
> 
> > Do you mean I should add Inkscape's python distribution to my PATH?
> 
> @pglpm No, inkscape should handle it on it's own.

Indeed, Jessyink seems not to see some python files that are there, either. I'll submit this bug to Inkscape. Cheers.

**Comment by jcwinkler** [comment]
See my comments in #203

**Comment by pglpm** [comment]
Confirmed fixed, as in [#203](https://github.com/textext/textext/issues/203)
